### PR TITLE
Fixing reference to TriggerLike

### PR DESCRIPTION
**Pull request by joshuaskelly**
## Summary
Fixes dangling reference to `TriggerLike`

### CODE REVIEW

File: jsonschema/current/Interfaces.schema.json [hunk 1]
```diff
@@ -17,13 +17,9 @@
                 {"$ref": "dungeoneer/entities/Sprite.schema.json"},
                 {"$ref": "dungeoneer/entities/projectiles/Projectile.schema.json"},
                 {"$ref": "dungeoneer/entities/triggers/BasicTrigger.schema.json"},
```
Comment (PythooonUser) [inline jsonschema/current/Interfaces.schema.json:19]: While you are at it, what do you think of grouping these `BasicTrigger` related triggers into a `BasicTriggerLike` group?

**Review by PythooonUser — APPROVED** [review]
Looking good!

File: jsonschema/current/Interfaces.schema.json [hunk 2]
```diff
@@ -97,6 +93,15 @@
             "oneOf": [
                 {"$ref": "dungeoneer/statuseffects/StatusEffect.schema.json" }
             ]
+        },
+        "TriggerLike": {
+            "oneOf": [
+                {"$ref": "dungeoneer/entities/triggers/ButtonModel.schema.json"},
```
Comment (PythooonUser) [inline jsonschema/current/Interfaces.schema.json:99]: I'm confused a bit. `ButtonModel` does not inherit from `Trigger`. So e.g. `Actor::useTrigger` would be wrong.

Comment (joshuaskelly) [reply jsonschema/current/Interfaces.schema.json]: Thanks for catching this. Fixed!

File: jsonschema/current/Interfaces.schema.json (hunk 1 above)
Comment (PythooonUser) [reply jsonschema/current/Interfaces.schema.json:19]: As discussed, not used at the moment. So we can skip it.

**Pull request merged**